src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -53,10 +57,9 @@ const useSaveAnonChatHook = () => {
 
     initAnonymousChat.message.cid = initAnonymousChat.message.cid.replace('messaging:', '');
 
-    const chat = ChatSchema.fromInitAnonymousChatAPI(initAnonymousChat, status);
-    await chat.save(localDb);
-
     try {
+      const chat = ChatSchema.fromInitAnonymousChatAPI(initAnonymousChat, status);
+      await chat.save(localDb);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on adding error handling here. However, similar to my previous comment, consider providing user feedback or implementing retry logic instead of just logging the error.



<!-- This is an auto-generated comment by OSS CodeRabbit -->